note.py

```diff
@@ -427,14 +427,14 @@ def __create_widgets(self, icons):
         self.commandframe = ttk.Frame(self)
         self.new_button = ttk.Button(self.commandframe, image=icons.new, command=self.model.add_new)
         self.new_button.pack(side = tk.RIGHT, fill=tk.X)
-        ToolTip(self.new_button, msg="add new note", delay=1.0)
+        ToolTip(self.new_button, msg="add new note\nctrl + n", delay=1.0)
```

Comment:
I like the idea, but it is common to use capitalized names for accelerators.

```suggestion
        ToolTip(self.new_button, msg="add new note (Ctrl+N)", delay=1.0)
```